husky_components/overlay/main.js

```diff
@@ -682,6 +682,11 @@ define([], function() {
             this.overlay.slides[slide].$content = this.sandbox.dom.find(constants.contentSelector, this.overlay.slides[slide].$el);
             this.overlay.slides[slide].$header = this.sandbox.dom.find(constants.headerSelector, this.overlay.slides[slide].$el);
 
+            if(this.slides[slide].languageChanger !== null || this.slides[slide].tabs !== null){
+                this.overlay.slides[slide].$subheader = this.sandbox.dom.createElement('<div class="' + constants.subheaderClass + '"/>');
+                this.sandbox.dom.after(this.overlay.slides[slide].$header, this.overlay.slides[slide].$subheader);
```

Comment:
The jQuery abstraction should not be done anymore. The new code we write is allowed to directly call the jQuery functions. In this example:
`this.overlay.slides[slide].$header.after(this.overlay.slides[slide].$subheader)`